<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>s21 Christmas Tree</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="base_tree_image">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>551</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;://base_tree.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_8">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_11">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_12">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_13">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_14">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_15">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_16">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_17">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_18">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_19">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_20">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>510</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_21">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="toy_slot_22">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: grey;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Slot" name="star_slot">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;://star_logo.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="toy_bg_switcher">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Фон слотов</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Slot</class>
   <extends>QLabel</extends>
   <header>slot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
